src/js/edu-benefits/routes.jsx

```diff
@@ -13,68 +15,83 @@ const routes = [
+      path="/veteran-information/secondary-contact"
+      chapter={chapters.veteranInformation}
+      name="Secondary Contact"/>,
   <Route
       component={PlaceholderSection}
       key="/veteran-information/direct-deposit"
-      path="/veteran-information/direct-deposit"/>,
+      path="/veteran-information/direct-deposit"
+      chapter={chapters.veteranInformation}
+      name="Direct Deposit"/>,
   <Route
       component={BenefitsSelection}
       key="/benefits-eligibility/benefits-selection"
-      path="/benefits-eligibility/benefits-selection"/>,
+      path="/benefits-eligibility/benefits-selection"
+      chapter={chapters.benefitsEligibility}/>,
   <Route
       component={PlaceholderSection}
       key="/military-history/military-service"
-      path="/military-history/military-service"/>,
-  <Route
-      component={PlaceholderSection}
-      key="/military-history/additional-information"
-      path="/military-history/additional-information"/>,
+      path="/military-history/military-service"
+      chapter={chapters.militaryHistory}
+      name="Military Service"/>,
   <Route
       component={PlaceholderSection}
       key="/military-history/rotc-history"
-      path="/military-history/rotc-history"/>,
+      path="/military-history/rotc-history"
+      chapter={chapters.militaryHistory}
+      name="ROTC History"/>,
   <Route
       component={PlaceholderSection}
       key="/military-history/benefits-history"
-      path="/military-history/benefits-history"/>,
+      path="/military-history/benefits-history"
+      chapter={chapters.militaryHistory}
+      name="Benefits History"/>,
   <Route
       component={PlaceholderSection}
-      key="/education-history/education-information"
-      path="/education-history/education-information"/>,
+      key="/military-history/dependents"
+      path="/military-history/dependents"
+      chapter={chapters.militaryHistory}
+      name="Dependents"/>,
   <Route
       component={PlaceholderSection}
-      key="/education-history/additional-information"
-      path="/education-history/additional-information"/>,
+      key="/education-history/education-information"
+      path="/education-history/education-information"
+      chapter={chapters.educationHistory}/>,
   <Route
       component={PlaceholderSection}
       key="/employment-history/employment-information"
-      path="/employment-history/employment-information"/>,
+      path="/employment-history/employment-information"
+      chapter={chapters.employmentHistory}/>,
   <Route
       component={PlaceholderSection}
       key="/school-selection/school-information"
-      path="/school-selection/school-information"/>,
+      path="/school-selection/school-information"
+      chapter={chapters.schoolSelection}/>,
   // Review and Submit route.
   <Route
       component={PlaceholderSection}
       key="/review-and-submit"
-      path="/review-and-submit"/>,
+      path="/review-and-submit"
+      chapter={chapters.review}/>,
```

Comment:
Part of me wants to write this in a DRY-er way.  But, I think this reads well, and additional magic wouldn't improve that.

Speaking of readability, I think a comment describing what Chapters are could be useful.